apps/server/src/test/testData/institution.ts

```diff
@@ -61,3 +63,60 @@ export const finicityInsitutionData = {
     }
   }
 }
+
+export class LocalInstitutionFactory {
```

Comment:
Why does this live in a testData file?

Why are we adding "Local" to the name? Don't we expect the institutions to look the same regardless of whether they're in our local default file or coming from the API?